SPECS/mariner-repos/mariner-repos.spec

```diff
@@ -30,16 +31,29 @@ Requires:   %{name} = %{version}-%{release}
 %description preview
 %{summary}
 
+%package extras
```

Comment:
I see we also have a preview repo under `[https://packages.microsoft.com/cbl-mariner/$releasever/preview/extras/`](https://packages.microsoft.com/cbl-mariner/$releasever/preview/extras/`), so we should include it as well.